bundles/AdminBundle/src/EventListener/TwoFactorListener.php

```diff
@@ -51,7 +50,6 @@ public function __construct(TwoFactorProviderRegistry $providerRegistry, Prepara
     public function onAuthenticationComplete(TwoFactorAuthenticationEvent $event)
     {
         // this session flag is set in \Pimcore\Bundle\AdminBundle\Security\AdminAuthenticator
```

Comment:
```suggestion
        // this session flag is set in \Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminAbstractAuthenticator
```
but wondering are we able to solve this TODO?

Reply:
I can't think of a better solution than using a session attribute at the moment.

Reply:
ah ok, then no problem, i also don't have a better idea.
Then it just missing this change in the comment and the rest looks good to me, ready to be merged